packages/load/src/load-typedefs.ts

```diff
@@ -151,3 +145,17 @@ function prepareResult({
   debugTimerEnd('@graphql-tools/load: prepareResult');
   return sortedResult;
 }
+
+export class NoTypeDefinitionsFound extends Error {
+  constructor(pointerList: string[]) {
+    const rows: string[] = [];
+    rows.push('Unable to find any GraphQL type definitions for the following pointers:');
+    pointerList.forEach(pointer => {
+      rows.push(`- ${pointer}`);
+    });
+    const message = rows.join('\n');
```

Comment:
I'm joining strings this way because I find the template literal adds a lot of extra spacing on the left of the terminal.